### ISSUE TITLE
Open to Contributions by PR?

### ISSUE THREAD

**Issue by quadrophobiac** [markwk/awesome-biomarkers#1]
Hello

I think this repo is a good idea - are you open to additions through pull request?

**Comment by markwk** [comment]
Of course! I'm actually keen to get back to working on this myself, especially in terms of building something on top of the data itself.

**Comment by aaronkurz** [comment]
@markwk I would love to help out and join on an open source BiomarkerDB software!

**Comment by Resram** [comment]
Hi Mark,
Will get back to you soon for a discussion.
Thank you
Reshmi
